test/TestLoopFuser.cxx

```diff
@@ -78,12 +78,12 @@ GPU_TEST(TestPacker, packFixedRange) {
 
    // pack has not been flushed, so
    // host data should not be updated yet
-   int * host_dst = dst.getPointer(care::CPU, false);
-   int * host_src = src.getPointer(care::CPU, false);
+   const int* host_src = src.data(chai::CPU, false);
+   const int* host_dst = dst.data(chai::CPU, false);
 
-   for (int i = 0; i < 2; ++i) {
-      EXPECT_EQ(host_dst[i], -1);
-      EXPECT_EQ(host_src[i], i);
+   for (int i = 0; i < arrSize; ++i) {
```

Comment:
this was only checking a small bit before?

Reply:
Yeah, I guess so.

Reply:
Probably because of the huge spam on failure :)

Reply:
Good catch!